tests/voq/test_fabric_reach.py

```diff
@@ -138,8 +134,11 @@ def test_fabric_reach_linecards(duthosts, enum_frontend_dut_hostname,
 
             remoteSlot = int(referencePortData['peer slot'])
             remoteAsic = int(referencePortData['peer asic'])
-            remoteMod = supervisorAsicBase + (remoteSlot - 1)*2 + remoteAsic
-            referenceRemoteModule = str(remoteMod)
+            remoteMod = (remoteSlot - 1)*2 + remoteAsic
+            # Get the fabric switch ID from config DB
+            referenceRemoteModule = supervisor.shell(
```

Comment:
can we cache this data iso running the cli for every line to make the test run faster?